adaptivemd/scheduler.py

```diff
@@ -362,7 +362,7 @@ def replace_prefix(self, path):
         path = path.replace('sandbox://', '../..')
 
         # the main remote shared FS
-        path = path.replace('shared://', '../../..')
+        path = path.replace('shared://', '')
```

Comment:
Sorry, I got the concept of `shared://` wrong. Change undone and updated docs. 

Just for the record: The `File` prefixes such as `shared://` are explained in the `File` docs.